massa-client/src/cmds.rs

```diff
@@ -989,29 +1023,179 @@ impl Command {
                     Err(e) => rpc_error!(e),
                 }
             }
-            Command::node_whitelist => {
-                let ips = parse_vec::<IpAddr>(parameters)?;
-                match client.private.node_whitelist(ips).await {
-                    Ok(()) => {
-                        if !json {
-                            println!("Request of whitelisting successfully sent!")
-                        }
+            Command::node_bootsrap_blacklist => {
+                if parameters.is_empty() {
+                    match client.private.node_bootstrap_blacklist().await {
+                        Ok(bootsraplist_ips) => Ok(Box::new(bootsraplist_ips)),
+                        Err(e) => rpc_error!(e),
                     }
-                    Err(e) => rpc_error!(e),
+                } else {
+                    let cli_op = match parameters[0].parse::<CLIOperation>() {
+                        Ok(op) => op,
+                        Err(_) => bail!(
+                            "failed to parse operation, supported operations are: [add, remove]"
+                        ),
+                    };
+                    let args = &parameters[1..];
+                    if args.is_empty() {
+                        bail!("[IpAddr] parameter shouldn't be empty");
+                    }
+                    let ips = parse_vec::<IpAddr>(args)?;
+                    let res: Result<Box<dyn Output>> = match cli_op {
+                        CLIOperation::Add => {
+                            match client.private.node_add_to_bootstrap_blacklist(ips).await {
+                                Ok(()) => {
+                                    if !json {
+                                        println!(
+                                            "Request of bootsrap blacklisting successfully sent!"
+                                        )
+                                    }
+                                    Ok(Box::new(()))
+                                }
+                                Err(e) => rpc_error!(e),
+                            }
+                        }
+                        CLIOperation::Remove => {
+                            match client
+                                .private
+                                .node_remove_from_bootstrap_blacklist(ips)
+                                .await
+                            {
+                                Ok(()) => {
+                                    if !json {
+                                        println!("Request of remove from bootsrap blacklist successfully sent!")
+                                    }
+                                    Ok(Box::new(()))
+                                }
+                                Err(e) => rpc_error!(e),
+                            }
+                        }
+                        _ => bail!("Not implemented"),
```

Comment:
Let's match `AllowAll` explicitly and improve this error message, also in `node_peers_whitelist`

Reply:
It was my first implementation. But I changed in case we add a new Operation to ListOperation we don't have ton propagate to match arms.

Reply:
We'd want to propagate anyway, if someone misuses the command I expect it to retrieve a proper error not juste a "Not implemented"

Reply:
Checked and it's now fixed; closing it